.github/workflows/main.yaml

```diff
@@ -6,11 +6,13 @@ on:
     paths-ignore:
       - "release-index.yaml"
       - "replica-releases/**"
+      - "release-controller/**"
```

Comment:
should we also add a dependency for the python requirements - a change in the lock files could break it.